### DISCUSSION TITLE
how to perform batch job for heat map generation?

### DISCUSSION THREAD

**Discussion by AdanRott** [OpenChrom/openchrom#528] [Q&A] [answered]

**Comment by Mailaender** [comment, marked as answer]
Hi, currently you can only enable in the toolbar `Chromatogram → Show Heatmap` and look at it. There is no way to save it, and it is also not part of the process methods, so you can't use it in the batch jobs.

**Reply by AdanRott** [reply]
Thank you.

**Reply by lorenzgerber** [reply]
Probably the quickest solution at the moment would be to use the OpenChrom batch job feature for exporting the chromatogram data into csv files. Then you could use an open source tool/language such as R(https://cran.r-project.org/) to batch plot heatmaps.
